presto-docs/src/main/sphinx/release/release-0.183.rst

```diff
@@ -0,0 +1,57 @@
+=============
+Release 0.183
+=============
+
+General Changes
+---------------
+
+* Fix planning failure for queries that use ``GROUPING`` and contain aggregation expressions
+  that require implicit coercions.
+* Fix memory over-accounting for stages where aggregation is followed by other non-trivial work.
+* Fix formatting in EXPLAIN ANALYZE output.
+* Fix planning error for `LEFT JOIN` converted to `INNER JOIN` without equi-conditions.
+* Improve error message when query contains unsupported correlated subquery.
+* Improve performance of ``CAST(json_parse(...) AS ...)``
+* Add :func:`map_from_entries` and :func:`map_entries` function
+* Make possible for `CHAR` functions (like `subst`) to  have empty spaces at the end of
+  the result.
+* Spill to disk for aggregation uses revocable memory. Now spilling takes place only if
+  query runs out of cluster memory during execution.
+* Remove `experimental.operator-memory-limit-before-spill` config property and
+  `operator_memory_limit_before_spill` session property.
+* Add `experimental.aggregation-operator-unspill-memory-limit` config property and
+  `aggregation_operator_unspill_memory_limit` session property. Properties specify how much
+  memory can be used for merging spilled aggregation data from disk.
+
+Hive Changes
+------------
+
+* Work around `ORC-222 <https://issues.apache.org/jira/browse/ORC-222>`_ which results in invalid
+  summary statistics in ORC files when the input data contains invalid string data.
+  Normally when this occurs the query fails, but in rare situations incorrect results
```

Comment:
```
  Previously, this would usually cause the query to fail, but in rare cases it could
  cause wrong results by incorrectly skipping data based on the invalid statistics.